src/atoms/cc-select.js

```diff
@@ -0,0 +1,217 @@
+import { css, html, LitElement } from 'lit-element';
+import { classMap } from 'lit-html/directives/class-map.js';
+
+import { dispatchCustomEvent } from '../lib/events.js';
+import { i18n } from '../lib/i18n.js';
+
+/**
+ * @typedef {import('./types.js').Option} Option
+ */
+
+/**
+ * A component wrapping native select element and its label.
+ *
+ * @cssdisplay inline-block
+ *
+ * @event {CustomEvent<string>} cc-select:input - Fires the `value` whenever the `value` changes.
+ *
+ * @slot error - The error message to be displayed right below the select element (low margin + red text)
+ */
+export class CcSelect extends LitElement {
+  static get properties () {
+    return {
+      disabled: { type: Boolean, reflect: true },
+      /** @required */
+      label: { type: String },
+      name: { type: String, reflect: true },
+      options: { type: Array },
+      placeholder: { type: String },
+      required: { type: Boolean },
+      value: { type: String },
+      _uniqueName: { type: String, attribute: false },
+      _uniqueId: { type: String, attribute: false },
```

Comment:
Sorry, I think my last comment about those was confusing.

I think:
* `_uniqueName` should be `_uniqueId` (used for the component and its label)
* `_uniqueId` should be `_uniqueErrorId` (used for the error message linked to the component)